src/EventsPage/components/AllUpcomingEvents.js

```diff
@@ -28,6 +32,7 @@ export default (props) => {
 
 const renderEventList = (events) => {
 	return events.map(event => {
+		if (event.date < today) {
 		return (
```

Comment:
I think this whole return statement still needs to be indented once to nest under the `if` block.